app/javascript/chat/chat.jsx

```diff
@@ -453,6 +453,7 @@ export default class Chat extends Component {
     }
     let newMessages = [];
     if (
+      activeChannelId &&
```

Comment:
Can you explain the hotfix a bit in the description of the PR? Thanks!

Reply:
Done !